opm/input/eclipse/EclipseState/Tables/SimpleTable.cpp

```diff
@@ -16,34 +16,41 @@
   You should have received a copy of the GNU General Public License
   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
  */
-#include <utility>
-#include <iostream>
 
 #include <opm/input/eclipse/EclipseState/Tables/SimpleTable.hpp>
+
 #include <opm/input/eclipse/EclipseState/Tables/TableSchema.hpp>
+
 #include <opm/input/eclipse/Deck/DeckItem.hpp>
 
+#include <stdexcept>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include <iostream>
```

Comment:
order

Reply:
> order

Yeah... To be honest I wanted to remove `<iostream>` altogether, but we use `std::cerr` in [`assertJFuncPressure()`](https://github.com/OPM/opm-common/blob/c780f384e7c5fa73979a03809b7524df463be23c/opm/input/eclipse/EclipseState/Tables/SimpleTable.cpp#L175) so that'll have to be follow-up work.